src/line_launcher/mod.rs

```diff
@@ -374,13 +418,16 @@ impl LineLauncher {
         }
     }
 
+    #[allow(clippy::too_many_arguments)]
     fn possibly_schedule_note_on(
         &self,
         playing_state: &mut PlayingState,
         beat_message: BeatNumber,
         schedule_note_on_sender: &Sender<ScheduleNoteOnMessage>,
         duration_between_sixteenth_notes: &DurationBetweenSixteenthNotes,
         ahead_or_behind_the_beat_ratio: &Option<f64>,
+        randomize_note_start_time_ratio: &Option<f64>,
```

Comment:
Since an `Option<f64>` can be copied, I don't think there's any point in using a reference to it. (Passing an option as an argument to this function won't move it, just copy it.)